From 2815e43e67ebc8adb77b9556b64d97fc55b1d6ae Mon Sep 17 00:00:00 2001
From: Daniel Oom <oom.daniel@gmail.com>
Date: Mon, 14 Aug 2017 14:05:10 +0200
Subject: [PATCH 1/4] Remove ATM numpad reordering

---
 kbddvp.c | 51 ---------------------------------------------------
 1 file changed, 51 deletions(-)

diff --git a/kbddvp.c b/kbddvp.c
index e3522d0..699fed7 100644
--- a/kbddvp.c
+++ b/kbddvp.c
@@ -249,7 +249,6 @@ static LDATA VK_TO_WCHARS3 numPad[] = {
         { VK_MULTIPLY,  0,      '*',    ')',    ')'             },
         { VK_SUBTRACT,  0,      '-',    '$',    '$'             },
         //-----------------------------------------------
-#ifdef NO_ATM
         { VK_NUMPAD7,   0,      '7',    'a',    'A'             },
         { VK_NUMPAD8,   0,      '8',    'b',    'B'             },
         { VK_NUMPAD9,   0,      '9',    'c',    'C'             },
@@ -263,21 +262,6 @@ static LDATA VK_TO_WCHARS3 numPad[] = {
         { VK_NUMPAD1,   0,      '1',    '=',    '='             },
         { VK_NUMPAD2,   0,      '2',    'x',    'x'             },
         { VK_NUMPAD3,   0,      '3',    ':',    ':'             },
-#else
-        { VK_NUMPAD1,   0,      '1',    'a',    'A'             },
-        { VK_NUMPAD2,   0,      '2',    'b',    'B'             },
-        { VK_NUMPAD3,   0,      '3',    'c',    'C'             },
-        //-----------------------------------------------
-        { VK_NUMPAD4,   0,      '4',    'd',    'D'             },
-        { VK_NUMPAD5,   0,      '5',    'e',    'E'             },
-        { VK_NUMPAD6,   0,      '6',    'f',    'F'             },
-        { VK_ADD,       0,      '+',    ',',    ','             },
-        { VK_ABNT_C2,   0,      '+',    ',',    ','             },
-        //-----------------------------------------------
-        { VK_NUMPAD7,   0,      '7',    '=',    '='             },
-        { VK_NUMPAD8,   0,      '8',    'x',    'x'             },
-        { VK_NUMPAD9,   0,      '9',    ':',    ':'             },
-#endif
         //-----------------------------------------------
         { VK_NUMPAD0,   0,      '0',    '0',    '\\'            },
         { VK_DECIMAL,   0,      '.',    ';',    WCH_NONE        },
@@ -406,7 +390,6 @@ static LDATA USHORT scanCodesToVirtualKeys[] = {
         // not on the 0x47 scan code. note however that Alt+NumPad to enter the
         // ASCII code of a character ALWAYS uses the scan code, no matter the
         // assignment here.
-#ifdef NO_ATM
         /* 0x47 */ VK_HOME      /* VK_NUMPAD7 */        | KBDNUMPAD     | KBDSPECIAL,
         /* 0x48 */ VK_UP        /* VK_NUMPAD8 */        | KBDNUMPAD     | KBDSPECIAL,
         /* 0x49 */ VK_PRIOR     /* VK_NUMPAD9 */        | KBDNUMPAD     | KBDSPECIAL,
@@ -420,24 +403,6 @@ static LDATA USHORT scanCodesToVirtualKeys[] = {
         /* 0x51 */ VK_NEXT      /* VK_NUMPAD3 */        | KBDNUMPAD     | KBDSPECIAL,
         /* 0x52 */ VK_INSERT    /* VK_NUMPAD0 */        | KBDNUMPAD     | KBDSPECIAL,
         /* 0x53 */ VK_DELETE    /* VK_DECIMAL */        | KBDNUMPAD     | KBDSPECIAL,
-#else
-        // when these codes are assigned directly VK_NUMPAD? then NumLock
-        // has no effect. however, I recommend that you leave it on since
-        // some applications test its state.
-        /* 0x47 */ VK_NUMPAD1,
-        /* 0x48 */ VK_NUMPAD2,
-        /* 0x49 */ VK_NUMPAD3,
-        /* 0x4A */ VK_SUBTRACT,
-        /* 0x4B */ VK_NUMPAD4,
-        /* 0x4C */ VK_NUMPAD5,
-        /* 0x4D */ VK_NUMPAD6,
-        /* 0x4E */ VK_ADD,
-        /* 0x4F */ VK_NUMPAD7,
-        /* 0x50 */ VK_NUMPAD8,
-        /* 0x51 */ VK_NUMPAD9,
-        /* 0x52 */ VK_NUMPAD0,
-        /* 0x53 */ VK_DECIMAL,
-#endif
         /* 0x54 */ VK_SNAPSHOT  | KBDEXT,
         /* 0x55 */ none,
         /* 0x56 */ VK_OEM_102,
@@ -1030,7 +995,6 @@ static LDATA VSC_LPWSTR keyNames[] = {
         0x46    /* VK_SCROLL            */,     L"Scroll Lock",
         // just as the code mapping is not specified with the VK_NUMPAD? symbols,
         // neither are the names (see the scanCodesToVirtualKeys table for details).
-#ifdef NO_ATM
         0x47    /* VK_HOME              */,     L"Numeric 7",
         0x48    /* VK_UP                */,     L"Numeric 8",
         0x49    /* VK_PRIOR             */,     L"Numeric 9",
@@ -1044,21 +1008,6 @@ static LDATA VSC_LPWSTR keyNames[] = {
         0x51    /* VK_NEXT              */,     L"Numeric 3",
         0x52    /* VK_INSERT            */,     L"Numeric 0",
         0x53    /* VK_DELETE            */,     L"Numeric Delete",
-#else
-        0x47    /* VK_NUMPAD1           */,     L"Numeric 1",
-        0x48    /* VK_NUMPAD2           */,     L"Numeric 2",
-        0x49    /* VK_NUMPAD3           */,     L"Numeric 3",
-        0x4A    /* VK_SUBTRACT          */,     L"Numeric -",
-        0x4B    /* VK_NUMPAD4           */,     L"Numeric 4",
-        0x4C    /* VK_NUMPAD5           */,     L"Numeric 5",
-        0x4D    /* VK_NUMPAD6           */,     L"Numeric 6",
-        0x4E    /* VK_ADD               */,     L"Numeric +",
-        0x4F    /* VK_NUMPAD7           */,     L"Numeric 7",
-        0x50    /* VK_NUMPAD8           */,     L"Numeric 8",
-        0x51    /* VK_NUMPAD9           */,     L"Numeric 9",
-        0x52    /* VK_NUMPAD0           */,     L"Numeric 0",
-        0x53    /* VK_DECIMAL           */,     L"Decimal Separator",
-#endif
         0x54    /* VK_SNAPSHOT          */,     L"System Request",
         0x56    /* VK_OEM_102           */,     L"Compose",
         0x57    /* VK_F11               */,     L"Function Key 11",
-- 
2.14.1.windows.1

